### ISSUE TITLE
OpenCV Java: Java crashes with Error 134 When Trying to Set Parameters on dnn.DetectionModel

### ISSUE THREAD

**Issue by nsadeh** [opencv/opencv#20520]
##### System information (version)

- OpenCV => 4.5..1-2 ('org.openpnp', name: 'opencv', version: '4.5.1-2')
- Operating System / Platform => macOS Catalina 10.15.7
- Compiler => Java Zulu 11

##### Steps to reproduce

1. Create a DetectionModel instance by calling `new DetectionModel(path, configPath)`
2. Set any parameter, e.g., `model.setInputParams(scale, size)
3. Java will crash on startup with error 134
4. 
##### Issue submission checklist

 - [x] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues,
       forum.opencv.org, Stack Overflow, etc and have not found solution
   
 - [x] I updated to latest OpenCV version and the issue is still there
   
 - [x] There is reproducer code and related data files: videos, images, onnx, etc

**Comment by nsadeh** [comment]
basically, detectionmodel is unuable currently in OpenCV Java.

**Comment by alalek** [comment]
> 4.5..1-2
> I updated to latest OpenCV version and the issue is still there

You need to update.

**Comment by nsadeh** [comment]
> > 4.5..1-2
> > I updated to latest OpenCV version and the issue is still there
> 
> You need to update.

The package I am using is the latest version available https://github.com/openpnp/opencv/releases

On the openCV website there is no mention of a different source of officially supported Java OpenCV on an artifactory like Maven, which makes it difficult to package as a Docker container with configuration-specificed dependency versioning.

**Comment by alalek** [comment]
There is no officially supported Maven repository.
You should ask for publishing of artifacts of newer releases in the mentioned repository.

**Comment by nsadeh** [comment]
Hey @alalek , I understand that there isn't an officially supported Maven repository and that I should ask the posters of the bundle I used. 
You didn't however explicitly acknowledge that the bug I reported was observed in earlier versions of opencv (e.g., 4.5.1, which I am using), although you did imply that upgrading will fix. Is this bug known and was it fixes between 4.5.1 and 4.5.3, or will I run into the same issues once I upgrade? Thank you.
